components/library/uwimap/Makefile

```diff
@@ -38,10 +40,11 @@ COMPONENT_FMRI=	library/uw-imap
 COMPONENT_SUMMARY=	University of Washington IMAP library
 COMPONENT_LICENSE=	Apache v2.0
 COMPONENT_LICENSE_FILE=	LICENSE.txt
+COMPONENT_BUILD_ARGS=	-j20
 
```

Comment:
Please remove this line. You should never put it into a Makefile but use an environment variable instead.